go.mod

```diff
@@ -19,10 +18,10 @@ require (
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/golang/snappy v0.0.1
 	github.com/google/go-cmp v0.5.5
-	github.com/influxdata/flux v0.120.1
+	github.com/influxdata/flux v0.123.1-0.20210729174141-a0db406d1fae
```

Comment:
Should we make a ticket to update to v0.124.0 now that it's out? Or do it as part of this PR so we don't lose track of it?

Reply:
Created https://github.com/influxdata/influxdb/issues/22066

Reply:
I don't want to update the PR here, I'd prefer to do a new PR to master